src/Process/CommandExecutor.php

```diff
@@ -43,4 +44,14 @@ public function passthru(string $command, OutputInterface $output): int
         });
         return $process->getExitCode();
     }
+
+    public function executeDirectly(string $command, bool $throwOnError = false): array
```

Comment:
I think it is nicer to return the Process. It is type-save and the Result can be got in a type-save manner.

```suggestion
    public function executeDirectly(string $command, bool $throwOnError = false): Process
```